graph/tags.go

```diff
@@ -105,9 +106,19 @@ func (store *TagStore) save() error {
 func (store *TagStore) reload() error {
 	f, err := os.Open(store.path)
 	if err != nil {
+		logrus.Debugf("Tag store file not found: %s", err)
```

Comment:
@LK4D4 its removed now. With this line have also removed the first test `TestDaemonStartWithNoRepoConf` as error message not show up in the logs.